backend/users/models/token.py

```diff
@@ -10,7 +10,7 @@
 
 class BearerToken(Token):
     note = models.TextField(null=True)
-    expires_at = models.DateTimeField(null=True)
+    expires_at = models.DateTimeField(null=False, default=timezone.now)
```

Comment:
I don't understand, does that mean that by default it expires instantly ?

Reply:
Thanks for asking, I am not sure I remember the rational behind this particular change.

Reply:
It may have been that all already existing tokens are revoked

Reply:
Do we really want a default value or do we want to force the user to provide it?

Reply:
What it means is that if no value is provided, the token will be created with instant expire. Note that the fronted will disable the possibility to create token with a null `expires_at` field. One other option is to remove the default ; meaning that creating a token with a null `expires_at` field will raise the following error:
```python
django.db.utils.IntegrityError: null value in column "expires_at" of relation "users_bearertoken" violates not-null constraint
```

Not sure of what would be the best behavior, but happy to rework this if we think it's worth.

Reply:
> Do we really want a default value or do we want to force the user to provide it

Did not refresh, sorry. Obviously better if we force the user to provide it indeed. Is there a clean way to do so?

Reply:
How did you get this `IntegrityError` ? The serializer should have returned an error when receiving an empty field for `expires_at`

Reply:
I have the error when running the [tests](https://github.com/Substra/substra-backend/blob/main/backend/api/tests/views/test_views_token.py#L15) and setting either:
* `required=True` in the `BearerTokenSerializer`
* `blank=False` in the `BearerToken` model